src/pages/v2/p/[projectId]/safe/index.page.tsx

```diff
@@ -33,7 +47,7 @@ export default function ProjectSafeDashboardPage({
         cv={cv}
       >
         <TransactionProvider>
-          <ProjectSafeDashboard />
+          <V2V3ProjectSafeDashboard projectId={projectId} />
```

Comment:
projectID is in `ProjectMetadataContext`, can't we use that instead?

Reply:
Done 🤤